pkg/validate/networking.go

```diff
@@ -172,9 +205,9 @@ func createPodSandboxWithPortMapping(c internalapi.RuntimeService, portMappings
 
 // checkDNSConfig checks the content of /etc/resolv.conf.
 func checkDNSConfig(c internalapi.RuntimeService, containerID string, expectedContent []string) {
-	By("get the content of /etc/resolv.conf via execSync")
-	cmd := []string{"cat", resolvConfigPath}
-	stdout, stderr, err := c.ExecSync(containerID, cmd, time.Duration(defaultExecSyncTimeout)*time.Second)
+	By("get the current dns config via execSync")
+	time.Sleep(100 * time.Second)
```

Comment:
What is this sleep for?  Is it for waiting some conditions?

Reply:
That's actually an artifact from when I was testing the port for this, thanks for pointing it out. I'll remove it